### ISSUE TITLE
[$1000] App resends copy pasted images from previous email replies along with new attachment reply

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15312]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app with user A
2. Open user B report and send any message
3. Open user B email and open notification email of user A message
4. Reply to message via email with copy pasted image (PrtSc button click and paste in windows and Shift+command+3 click on mac, right click on image to save to clipboard and paste), you can also send text with it if needed
5. Once message is recieved by user A, reply again to same email using user B email, now reply with any attachment image (you can use text too with it if needed)
6. Observe the message in user A

## Expected Result:
User A should only receive the latest sent attachment with text

## Actual Result:
User A receives both previous copy pasted image as well as latest sent attachment

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.2.74-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/220205513-1815b9c9-4976-4102-b9c8-49c39d784765.mp4

https://user-images.githubusercontent.com/43996225/220205565-c6be7721-a378-4e2c-9d64-e87f761c92b3.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @dhanashree-sawant
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1676917118596099

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0123f5c9ea5dc8a8d1</li>
        <li>Upwork Job ID: 1680605284573466624</li>
        <li>Last Price Increase: 2023-07-16</li>
    </ul>
</details>

**Comment by MelvinBot** [comment]
Triggered auto assignment to @MitchExpensify (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by ctkochan22** [comment]
Looks like this falls under a larger tracking issue for email replies/formatting. 

Pretty sure this is likely a Mailgun issue. If I'm guessing, we loop through the reply history. We usually omit most of it out, but in this case, maybe the image isn't filtered out and so we end up receiving two image. 

It would be interesting to try sending attachments twice. Or even a third time to see if the non-attachment image keeps appearing

**Comment by ctkochan22** [comment]
Lets put it on hold while we continue to consolidate email issues under this tracking gh

**Comment by ctkochan22** [comment]
Placed on hold. @MitchExpensify it would be great if you can still do the triage checklist 🙇‍♂️

**Comment by ctkochan22** [comment]
Still on hold for https://github.com/Expensify/App/issues/15336

**Comment by MelvinBot** [comment]
@MitchExpensify Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by MitchExpensify** [comment]
Waiting on new engineering volunteer for the parent issue: https://github.com/Expensify/App/issues/15336

**Comment by MitchExpensify** [comment]
> Waiting on new engineering volunteer for the parent issue: https://github.com/Expensify/App/issues/15336

Same as above

**Comment by MitchExpensify** [comment]
Same as above

**Comment by MitchExpensify** [comment]
Same as above

**Comment by MitchExpensify** [comment]
Still on hold

**Comment by MitchExpensify** [comment]
> Waiting on new engineering volunteer for the parent issue: https://github.com/Expensify/App/issues/15336

Same as above

**Comment by MitchExpensify** [comment]
_Same_

**Comment by MitchExpensify** [comment]
Same seeing as the dependent issue is not being pushed as a priority https://github.com/Expensify/App/issues/15336

**Comment by MitchExpensify** [comment]
_Same_

**Comment by MitchExpensify** [comment]
SAME

**Comment by MitchExpensify** [comment]
Same, still not a priority

**Comment by MitchExpensify** [comment]
You guessed it, still not a priority

**Comment by MitchExpensify** [comment]
Same

**Comment by MitchExpensify** [comment]
@dhanashree-sawant are you still able to reproduce this?

**Comment by dhanashree-sawant** [comment]
Hi @MitchExpensify, yes it is still reproducible on latest staging.

**Comment by MitchExpensify** [comment]
Reproduce too:

<img width="823" alt="image" src="https://github.com/Expensify/App/assets/36425901/731ed72f-102f-49c7-869a-6405594fcc55">

**Comment by MitchExpensify** [comment]
Still waiting on proposals, Melvin

**Comment by MitchExpensify** [comment]
Thanks @allroundexperts , made the change!

**Comment by MitchExpensify** [comment]
Labelling eng as this needs internal eng assignment

**Comment by marcochavezf** [comment]
No update yet

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update, focused on other 2 daily issues

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update, moving this issue to weekly

**Comment by marcochavezf** [comment]
No update, focused on TU

**Comment by MitchExpensify** [comment]
How are we looking here @marcochavezf? No worries if no update, wishing to keep Melvin off our backs 👍

**Comment by MitchExpensify** [comment]
> No update, focused on TU

This is the latest, Melvin

**Comment by marcochavezf** [comment]
No update yet here, focused on wave7

**Comment by MitchExpensify** [comment]
Same as above, Melvin

**Comment by marcochavezf** [comment]
No update, focused on other daily priorities and wave7

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update

**Comment by MitchExpensify** [comment]
I assume the same, @marcochavezf ?

**Comment by marcochavezf** [comment]
Yes sorry, still tackling a few other dailies

**Comment by marcochavezf** [comment]
No update

**Comment by MitchExpensify** [comment]
No update it seems, Melvin

**Comment by marcochavezf** [comment]
No update

**Comment by MitchExpensify** [comment]
Checkin in on this @marcochavezf, still on hold?

**Comment by marcochavezf** [comment]
Yeah, still focused on wave issues. Alternatively, do you think this should part of a vip project? In that case I can prioritize it

**Comment by MitchExpensify** [comment]
I'm not sure its part of any VIP project, no

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
Focused on wave issues, no update

**Comment by marcochavezf** [comment]
No update, but I think this GH can be a candidate for #vip-vsb

**Comment by marcochavezf** [comment]
No update, focused on wave work

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
Focused on wave work

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update, working on another weekly issue, but it's the next one on the list

**Comment by marcochavezf** [comment]
No update, focused on higher priority tasks/bugs

**Comment by MitchExpensify** [comment]
This is a chat centric issue so placing under VIP VSB as polish

**Comment by marcochavezf** [comment]
Focused on critical issues for spring release

**Comment by marcochavezf** [comment]
Spent a bit of time setting up the reply system locally

**Comment by MitchExpensify** [comment]
Not on hold or overdue, @marcochavezf is actively working on this

**Comment by MitchExpensify** [comment]
How are you doing here @marcochavezf ?

**Comment by marcochavezf** [comment]
Catching up from OOO, I will continue with it this week

**Comment by marcochavezf** [comment]
I will make a bit of time for this one today

**Comment by marcochavezf** [comment]
I am having issues with the configuration to receive the replies via email locally. I will move it to monthly since there are other pressing items, but feel free to change it again if disagree

**Comment by marcochavezf** [comment]
Same, no update here

**Comment by MitchExpensify** [comment]
Reassigning as I'm headed on paternity leave 🙇

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
Unnasigning myself to focus on other priorities

**Comment by JmillsExpensify** [comment]
Closing as something we won't fix, as it's not a priority now.
